README.md

```diff
@@ -181,43 +171,89 @@ For Example:
 
   export default function(assert) {
     return steps(assert)
-      .given('I add something to the context', function(next) {
+      .given('I add something to the context', function() {
         // Assign 'hello' to the variable outside the step chain
         something = 'hello';
         // Assign 'there' to a new variable in `this.ctx`
         this.ctx.something = 'there';
         assert.ok(true, this.step);
-        next();
       })
-      .then('it should be there in the next step', function(next) {
+      .then('it should be there in the next step', function() {
         // Do an assertion to check that 'there' has been passed correctly
         // to the next step
         assert.equal(this.ctx.something, 'there', this.step);
-        next();
       })
-      .then('external variable should be there in the next step', function(next){
+      .then('external variable should be there in the next step', function(){
         // Assert that the external variable still holds the information
         // we set in the first step
         assert.equal(something,'hello',this.step);
       });
   }
 ```
-##### Testing components that depend on other components
-If you need to test a component, that depends on other components, you can use the `@needs` gherkin decorator to specify what this component depends on.
 
-For example:
-```gherkin
-  @needs=component:ember-selectize
-  Feature: Component form-select
+### Annotations
+
+You already saw the use of the `@setupApplicationTest` annotation in the example feature file above.
+Yadda's [support for annotations](https://acuminous.gitbooks.io/yadda-user-guide/en/feature-specs/annotations.html`) can
+be used to customize the way tests are run.
+
+The implementation for the way certain annotations affect your tests lives in the `tests/yadda-annotations.js` file. 
+The addon installs a this file with a default implementation as described below, but you can freely customize it at your
```

Comment:
Small typo - extra 'a': 

...installs a this...

Reply:
Fixed